<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuizScreen2</class>
 <widget class="QWidget" name="QuizScreen2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>630</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="quizLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>671</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Module 2 Quiz: What are the Types of Avalanches?</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>1151</width>
     <height>541</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-142</y>
      <width>1135</width>
      <height>780</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>780</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>697</width>
       <height>754</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0">
      <item>
       <widget class="QLabel" name="q1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>17</pointsize>
         </font>
        </property>
        <property name="text">
         <string>What is true about loose snow avalanches?</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="a1">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>They are usually caused by rainfall</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question1</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="b1">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>They are characterized by a solid snowpack</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question1</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="c1">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>They are common on steep slopes after a fresh snowfall</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question1</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="d1">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>They occur only during the winter months</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question1</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="q2">
        <property name="font">
         <font>
          <pointsize>17</pointsize>
         </font>
        </property>
        <property name="text">
         <string>What distinguishes a slab avalanche from the other types?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="a2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>It has a single point of origin</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="b2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>It is characterized by loose snow</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="c2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>It only occurs on relatively flat terrain</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="d2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>In involves the movement of a large block of ice</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="q3">
        <property name="font">
         <font>
          <pointsize>17</pointsize>
         </font>
        </property>
        <property name="text">
         <string>What is the primary characteristic of powder snow avalanches?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="a3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>They consist only of loose snow</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question3</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="b3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>They travel at a relatively slow speed</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question3</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="c3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>They comprise a mix of loose snow and slabs</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question3</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="d3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>They are most common during warmer months</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question3</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="q4">
        <property name="font">
         <font>
          <pointsize>17</pointsize>
         </font>
        </property>
        <property name="text">
         <string>What is the leading cause of wet snow avalanches?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="a4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Precipitation during the summer months</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question4</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="b4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Intense sunlight</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question4</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="c4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Friction from steep slopes</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question4</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="d4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Strong winds</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question4</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>630</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Go Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>630</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Module</string>
   </property>
  </widget>
  <widget class="QLabel" name="submitMessage">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>20</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="question1"/>
  <buttongroup name="question3"/>
  <buttongroup name="question4"/>
  <buttongroup name="question2"/>
 </buttongroups>
</ui>
